components/GraphDiagram/GraphDiagram.tsx

```diff
@@ -92,11 +92,11 @@ export default function GraphDiagram({ activity }: { activity: LoadActivity }) {
   async function handleGraphClick(event: React.MouseEvent) {
     const target = event.target as HTMLDivElement;
 
-    if ($('#graph-controls').contains(target)) return;
+    if ($('#graph-controls')!.contains(target)) return;
 
-    const el = $.up<SVGElement>(target, '.node');
+    const el = target.closest('g.node');
 
-    const moduleKey = el ? $(el, 'title')?.textContent?.trim() : '';
+    const moduleKey = el ? el.querySelector('title')?.textContent?.trim() : '';
```

Comment:
The new `$` helper is a simple bound `document.querySelector` call, so it can't be used in these cases. 

If you'd like, I can restore the previous `$(TYPE, PARENT)` signature. I actually have a similar well-typed module for this, but I haven't used it:

https://github.com/fregante/select-dom/blob/afdd637791737fc9dbcc8590d6655246d16bb4d5/index.ts#L13-L36

Reply:
Yeah, let's go with `select-dom` so we can have consistent use of $() and $(), throughout.

(Will that let us remove the `typed-query-selector` dependency if we're no longer calling querySelector/All directly?)

Reply:
> remove the `typed-query-selector` dependency

You still want it for the .closest calls